app/stories/pages/index.stories.tsx

```diff
@@ -2,11 +2,18 @@ import { ComponentMeta, ComponentStory } from '@storybook/react'
 
 import IndexPage from '@pages/index'
 
+import { getForwardRoute } from '@utils/routing'
+
+import { getEmptyMockSession } from '../../tests/helpers/mockData'
+
 export default {
   title: 'Pages/Index',
   component: IndexPage,
 } as ComponentMeta<typeof IndexPage>
 
-const Template: ComponentStory<typeof IndexPage> = () => <IndexPage />
+const Template: ComponentStory<typeof IndexPage> = (args) => {
+  args.forwardRoute = getForwardRoute('/', false, getEmptyMockSession())
```

Comment:
Notes for reviewers: In the storybook story for this page, we mock `forwardRoute` by calling `getForwardRoute()` directly. In the application, `_app.tsx` would handle making this call. `getForwardRoute()` has the following function signature:

```ts
function getForwardRoute(
  pathname: string,
  reviewMode: boolean,
  session: SessionData | ((value: SessionData) => void)
)
```

Reply:
This just means that storybook will robustly tell you what the action button will go to (when you click on the `Actions` tab in storybook and then click the "Get started" button), but I'm not expecting it to be the thing that actually tests this functionality.

<img width="642" alt="Screen Shot 2022-10-28 at 3 05 34 PM" src="https://user-images.githubusercontent.com/67701/198740463-93b85e2a-36d2-43ce-bf6c-1ef63517bc30.png">

Reply:
Wanted to say I read this over and haven't done something like this in storybook before so I do not have too much wisdom to offer 🙏